src/main/scala/org/labrad/manager/Listener.scala

```diff
@@ -101,7 +106,7 @@ extends Logging {
              p.addLast(new SniHandler(tlsHostConfig.sslCtxs))
            }
            p.addLast("packetCodec", new PacketCodec())
-           p.addLast("loginHandler",
+           p.addLast(loginGroup, "loginHandler",
```

Comment:
I wonder if it'd be helpful to add a quick comment that makes it clear that this loginGroup implies a separate thread, just so that it's clear which parts of this code execute on different threads.